### ISSUE TITLE
PayPal auth popup input fields issue (FB webview on iOS)

### ISSUE THREAD

**Issue by dmalis** [paypal/paypal-checkout#403]
### Description
When trying to enter email and/or password on PP auth popup the typing cursor starts jumping around the screen - in other words, cursor doesn't stay within the input field/s. 

- [x] Screenshots or videos that show the issue
![pp-checkout-fb-bug](https://user-images.githubusercontent.com/274254/28120473-6b05690e-6718-11e7-80ed-76ac4ac548d8.jpg)

- [x] Your code
```
    paypal.Button.render({
        env: window.env,
        style: {
            label: 'checkout', // checkout || credit
            size: 'medium', // tiny | small | medium
            shape: 'rect', // pill | rect
            color: 'blue' // gold | blue | silver
        },
        payment: function() {
            return paypalCheckoutInstance.createPayment({
                flow: 'checkout',
                amount: amount,
                currency: currency,
                enableShippingAddress: true
            });
        },
        onAuthorize: function(data, actions) {
            return paypalCheckoutInstance.tokenizePayment(data).then(function(payload) {
                //stuff
            });
        },
        onError: function(err) {
            console.error('checkout.js error', JSON.stringify(err, 0, 2));
            Raven.captureMessage('onError: ' + JSON.stringify(err, 0, 2));
        }
    }, '#paypal-button');
```

- [x] The version of checkout.js you're using (or the paypalobjects url)
checkout.4.0.89.min.js with breintree 3.19.1 

- [x] A link to your page with the issue, if possible
https://www.ulla.io

- [ ] All console logs during the time of the issue, especially error messages
- [ ] Does the issue also occur at [developer.paypal.com/demo/checkout](https://developer.paypal.com/demo/checkout)?

### Steps to reproduce
1. open FB app (latest) on iphone7 / ios 10.3.2
2. search for "ULLA smart hydration reminder"
3. open any link to www.ulla.io (eg. on first post)
4. add to cart > checkout > tap on "checkout with PP" button on shipping info page
4.1 PP popup opens
5. type in email and password
5.1 in case everything works, close the popup and try again

### Affected browsers

- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge
- [ ] IE
- [ ] Chrome Mobile/Tablet
- [ ] Safari Mobile/Tablet
- [x] Web View / Safari ViewController
- [ ] Other

**Comment by walmik** [comment]
On first glance it seems like an existing issue on iPhone 6 _presumably_ as per this link: https://discussions.apple.com/thread/7921428?start=0&tstart=0

In other words it may not be related directly to the Auth flow or the checkout flow itself. Will update this thread once we have some more definitive data.

**Comment by walmik** [comment]
It s also possible this issue could be a result of `overflow:auto` in the CSS as per https://muut.com/i/framework7/bugs-and-issues:input-field-curtor-cursor

**Comment by walmik** [comment]
From looking into it further, I m more inclined toward the thought that this issue is caused by `overflow:auto` on the modal DIV that holds the PayPal login page. Web auth certainly doesnt add this DIV. Either the button builder does that DIV or the ULLA page itself does that.

**Comment by dmalis** [comment]
I believe this style is not enforced by ULLA.IO site.

**Comment by bluepnume** [comment]
Still seeing this issue?

**Comment by vishakha94** [comment]
@walmik Are you still facing this issue?

**Comment by dmalis** [comment]
It works now.

**Comment by vishakha94** [comment]
Closing this issue.

**Comment by iranianpep** [comment]
We are experiencing the exact same issue. But just came across this: https://hackernoon.com/how-to-fix-the-ios-11-input-element-in-fixed-modals-bug-aaf66c7ba3f8

@dmalis did you need to apply any change to fix the issue or it's magically fixed?

**Comment by alex-mironov** [comment]
@iranianpep @dmalis have you guys managed to solve it?
Here is what happens in my case with React Native (sorry for slow animation) ![iospaypal](https://user-images.githubusercontent.com/2544767/43627399-edc19eaa-96fe-11e8-8b77-7111175fbbd2.gif)
